Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -328,7 +328,19 @@ void OnValidate()
         }
 
 #if UNITY_EDITOR
-        void AssignAssetID(string path) => m_AssetId = AssetDatabase.AssetPathToGUID(path);
+        void AssignAssetID(string path)
+        {
+            if (string.IsNullOrEmpty(path))
```

Comment:
please make this shorter:
if !nullorempty: assign assetid

Reply:
done